permissions/plugin-redis-notifier.yml

```
@@ -0,0 +1,7 @@
+---
+name: "redis-notifier"
+github: "jenkinsci/redis-notifier-plugin"
+paths:
+- "org/jenkins-ci/plugins/redis-notifier"
```

The value for paths should come from your pom.xml, it is the `<groupId>/<artifactId>` with periods replaced with slashes.

```suggestion
- "io/jenkins/plugins/redis-notifier"
```